docs/cli.mdx

```diff
@@ -0,0 +1,243 @@
+---
+title: Saleor CLI 
+---
+
+## Saleor CLI
```

Comment:
<img width="1164" alt="image" src="https://user-images.githubusercontent.com/5188636/166684619-fb2bd172-19cc-4fba-95c7-b2a68b61ec3a.png">

I believe this header can be skipped because it's a duplication of the page title.